jameda_interface_config.js

```diff
@@ -308,4 +308,36 @@ if (isMobileBrowser()) {
     ];
 }
 
+/**
+ * Checks browser if Firefox or Safari is used.
+ * @returns {?boolean}
+ */
+function isBrowserFirefoxOrSafari() {
```

Comment:
Maybe we should consider to use library like this: https://github.com/faisalman/ua-parser-js

Reply:
I actually came across in this library you're suggesting and in its description they have written >  However, it's not recommended to use this library as browser detection since the result may not be more accurate than using feature detection.

I know we can find other libraries, it is good to keep this in mind and for the next time to use a library instead of pure JS code to get these information. Hence, for this time I would like to move on with this PR and focus on other stuff what is more important.